pages/common/caller.md

```diff
@@ -0,0 +1,16 @@
+# caller
+
+> Print function context.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-caller>.
+
+- Print the line where the current function was called and its filename:
+
+`caller`
+
+- Print the line where the current function was called, its function name and filename:
+
+`caller 0`
+
+- Print the line where a frame above was called, its function name and filename:
+
+`caller {{1}}`
```

Comment:
> I need help with this page as I'm not sure if the language used here is entirely correct. I might be using words like "frame" wrong.

Maybe like this?

```suggestion
- Print the line starting the n'th-last function call, its function name and filename:

`caller {{n}}`
```

Reply:
I guess using technical words like frames won't help newbies either. I'll take that suggestion and massage it a bit.

Reply:
I think I got the wording in a good position now.

Reply:
agree :+1: